vault/expiration.go

```diff
@@ -2472,6 +2487,7 @@ func (m *ExpirationManager) markLeaseIrrevocable(ctx context.Context, le *leaseE
 	m.persistEntry(ctx, le)
 
 	m.irrevocable.Store(le.LeaseID, m.inMemoryLeaseInfo(le))
+	m.irrevocableLeaseCount++
```

Comment:
can you update the comment on line 2463 ("...no total lease count metrics/...") to be accurate given the new metrics?

Reply:
donzo df2931a